Formula/repo.rb

```diff
@@ -2,13 +2,15 @@ class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
   url "https://gerrit.googlesource.com/git-repo.git",
-      tag:      "v2.10",
-      revision: "352c93b68084e97ec7a3c09f0bd3f2a161290211"
+      tag:      "v2.11.1",
+      revision: "b64bec6acc0e2a835f427d8424af89dfcddaf08f"
   license "Apache-2.0"
   version_scheme 1
 
   bottle :unneeded
 
+  depends_on "python@3.9"
```

Comment:
Is this the right solution? Why isn't `repo` able to use the built-in python3?

Reply:
I missed the shebang update, just added.

Reply:
```diff
diff --git a/usr/local/Cellar/repo/2.11.1/bin/repo b/../repo.org
old mode 100755
new mode 100644
index d797bf4..8f13015
--- a/usr/local/Cellar/repo/2.11.1/bin/repo
+++ b/../repo.org
@@ -1,4 +1,4 @@
- #!/usr/local/opt/python@3.9/bin/python3
+ #!/usr/bin/env python
# -*- coding:utf-8 -*-
#
# Copyright (C) 2008 The Android Open Source Project
```